app/views/admin/project_steps/_modal_content.html.slim

```diff
@@ -69,11 +69,15 @@ section.step-fields class=(@mode == :show_and_form ? 'show-view' : 'edit-view')
           collection: ProjectStep.step_type_option_set.translated_list,
           include_blank: '---'
 
-      = f.input :parent_id, collection: @parents, label_method: :summary_or_none,
-        label: t("project_step.timeline_table.group"), include_blank: false
-
-      = f.input :schedule_parent_id, collection: @precedents, label_method: :summary_or_none,
-        label: t("project_step.timeline_table.preceding_step"), include_blank: true
+      = f.input :parent_id
+        .view-element = @step.parent.summary_or_none if @step.parent.present?
+        = f.input_field :parent_id, collection: @parents, label_method: :summary_or_none,
+          label: t("activerecord.attributes.project_step.parent"), include_blank: false
```

Comment:
@hooverlunch What is wrong here? I changed to activerecord translations and now it's broken. No matter what I put in the translation, it always says "parent". Even if I hard code the label....